commands/apps.go

```diff
@@ -331,59 +373,48 @@ func RunAppsCreateDeployment(c *CmdConfig) error {
 	return c.Display(displayers.Deployments{deployment})
 }
 
-// waitForAppDeploymentRunning waits for a app deployment to be running.
-func waitForAppDeploymentRunning(apps do.AppsService, appID string, deploymentID string) (*godo.Deployment, error) {
-	failCount := 0
+func waitForActiveDeployment(apps do.AppsService, appID string, deploymentID string) error {
+	const maxAttempts = 10 * 6 * 30
```

Comment:
Think we might want something a bit shorter here. 1800 attempts at 10 seconds each is 5 hours. Maybe something like 30 minutes?

Reply:
oh 30 minutes is more reasonable hahaha. Will do.